config/puma.rb

```diff
@@ -11,6 +11,12 @@
 # cause and implement a permanent solution.
 worker_timeout 600
 
+# Set the keepalive timeout higher than what nginx is set to
+# Defaults to 20
+# A shorter value on the Conjur side causes errors and
+# potential instability
```

Comment:
This comment could probably use a little more detail. What you wrote in the issue description was good, maybe incorporate some of that?

Reply:
I've updated the comment to better describe the rationale and what default value is being overridden.